scripts/ci/hermes/config.toml

```diff
@@ -68,3 +67,9 @@ gas_adjustment = 0.1
 clock_drift = '300s'
 trusting_period = '2days'
 trust_threshold = { numerator = '1', denominator = '3' }
+
+[chains.packet_filter]
+policy = 'deny'
+list = [
+  ['ica*', '*'], # deny relaying on all channels whose port starts with `ica`
```

Comment:
What is this about?

Reply:
we don't have  interchain accounts to relay in our CI, at least not now hopefully soon 😄